apps/cli/tf/bucket.tf

```diff
@@ -0,0 +1,70 @@
+module "acm_request_certificate" {
+  source = "cloudposse/acm-request-certificate/aws"
+
+  # https://github.com/cloudposse/terraform-aws-acm-request-certificate
+  version                           = "v0.17.0"
+  domain_name                       = "cli.sunodo.io"
+  process_domain_validation_options = true
+  ttl                               = "300"
+  zone_id                           = var.sunodo_cli_zone_id
+}
+
+module "cdn" {
+  source = "cloudposse/cloudfront-s3-cdn/aws"
+
+  # https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/tree/0.90.0
+  version                             = "v0.90.0"
+  name                                = "sunodo"
+  namespace                           = "cli"
+  stage                               = "prod"
+  aliases                             = ["cli.sunodo.io"]
+  dns_alias_enabled                   = true
+  parent_zone_id                      = var.sunodo_cli_zone_id
+  acm_certificate_arn                 = module.acm_request_certificate.arn
+  cloudfront_access_log_create_bucket = false
+  cloudfront_access_logging_enabled   = false
+  allowed_methods                     = ["GET", "HEAD", "OPTIONS"]
+  cached_methods                      = ["GET", "HEAD"]
+  versioning_enabled                  = false
+  block_origin_public_access_enabled  = false
+
+  depends_on = [module.acm_request_certificate]
+}
+
+data "aws_iam_policy_document" "sunodo_cli_s3_doc" {
+  statement {
+    actions = [
+      "s3:ListBucket",
+    ]
+    resources = [
+      "arn:aws:s3:::cli-prod-sunodo-origin",
+    ]
+  }
+  statement {
+    actions = [
+      "s3:*",
```

Comment:
Can this be more restrict/specific?

Reply:
I couldn't find any related docs in the oclif document. but, I'll check some restrictions.